Dockerfile

```diff
@@ -366,7 +365,6 @@ RUN LIBCUDA_FOUND=$(ldconfig -p | grep -v compat | awk '{print $1}' | grep libcu
             mkdir -p /opt/tritonserver/include && \
             cp -r server/install/bin /opt/tritonserver/. && \
             cp -r server/install/lib /opt/tritonserver/. && \
-            cp -r server/install/backends /opt/tritonserver/. && \
```

Comment:
Remember to make the necessary changes in build_variants test as well.

Reply:
Right, I should change those tests too.